blender_exporter.py

```diff
@@ -0,0 +1,218 @@
+# The edanim exporter, exports skinned meshes to a special
+# format that is easy to parse and is written in plain text.
+#
+# This code is heavily based on the standard blender exporter
+# and gives a very handy way to export data.
+#
+# There is a lot of work left to do if this is to become a
+# "fully featured" exporter.
+#
+# The file format this generates, specifies what each line
+# does, currently its "geo:" for geometry data. "arm:" for
+# armature "anim:##name##:" is the animation with the name
+# "##name##"
```

Comment:
Dunno if you meant to skip this or forgot but I'll just bump it. Not a requirement.

Reply:
What did i supposedly forget in that code?